django/http/response.py

```diff
@@ -141,6 +137,16 @@ def charset(self):
     def charset(self, value):
         self._charset = value
 
+    @property
+    def reason_phrase(self):
+        if self._reason_phrase is not None:
```

Comment:
You don't set `self._reason_phrase` anywhere in here if the lookup fails. I don't think that's intended?